blob/driver/driver.go

```diff
@@ -66,6 +67,8 @@ type ObjectAttrs struct {
 	Size int64
 	// ContentType is the MIME type of the blob object. It must not be empty.
 	ContentType string
+	// ModTime is the modified time of the blob object.
```

Comment:
I can imagine circumstances where implementations might not have this information. Let's add a comment to the effect that `ModTime` might be the zero time if unknown.

Reply:
Updated comment in [c04be8e](https://github.com/google/go-cloud/pull/240/commits/c04be8e0a626e3709d36f16a7466fa4549969636).